Fix for #9608: RectangleSelector now reports mouse button press and release events correctly (#20611)
matplotlib/matplotlib: doc/api/next_api_changes/behavior/20611-MRR.rst

``RectangleSelector`` ``onselect`` now reports actual mouse press and release coordinates
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The current docstrings in ``RectangleSelector`` explain ``onselect`` as a function with the

> the signature ``def onselect(eclick: MouseEvent, erelease: MouseEvent)``. The old
> behaviour is to actually report the ``extent`` of the rectangle, not the mouse event
> coordinates. The new behaviour reports the actual mouse event coordinates.

Reviewer: ```suggestion
the signature ``def onselect(eclick: MouseEvent, erelease: MouseEvent)``. The
previous behaviour reported the ``extent`` of the rectangle, not the mouse event
coordinates. The new behaviour reports the actual mouse event coordinates.
```
